xblock_skytap/templates/skytap.html

```diff
@@ -0,0 +1,33 @@
+<h3>Launch exercise environment</h3>
+<form class="skytap-launch-form">
+  <!-- <div> -->
+  <!--   {% for user in users %} -->
+  <!--     <span>User: {{ user.name }} ({{user.email}}) -->
+  <!--   {% endfor %} -->
+  <!-- </div> -->
+  <div>
+    <label>
+      <span>Select your preferred keyboard layout:</span>
+      <select name="keyboard_layout">
+        <option value="nl-be">Dutch-Belgium</option>
+        <option value="uk">English (UK)</option>
+        <option value="us">English (US)</option>
+        <option value="fi">Finnish</option>
+        <option value="fr">French</option>
+        <option value="fr-be">French-Belgium</option>
+        <option value="fr-ch">French-Switzerland</option>
+        <option value="de">German</option>
+        <option value="de-ch">German-Switzerland</option>
+        <option value="is">Icelandic</option>
+        <option value="it">Italian</option>
+        <option value="jp">Japanese</option>
+        <option value="no">Norwegian</option>
+        <option value="pt">Polish</option>
+        <option value="es">Spanish</option>
```

Comment:
@bdero Nope, unfortunately not. From the API documentation (v1, v2) about the `vnc_keymap` setting:

> Keyboard language mapping. UTF-8 character type. For a list of supported international keyboards and attribute values, see [Setting an international keyboard layout for a VM](http://help.skytap.com/Setting_an_International_Keyboard_Layout.html). Attribute values are listed in parentheses on that page.